pom.xml

```diff
@@ -19,27 +20,27 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.66</version>
+        <version>4.75</version>
         <relativePath />
     </parent>
 
     <artifactId>fortify</artifactId>
-    <version>23.1.40-SNAPSHOT</version>
+    <version>${revision}${changelist}</version>
     <packaging>hpi</packaging>
 
     <properties>
-        <gitHubRepo>jenkinsci/fortify-plugin</gitHubRepo>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <revision>23.1.40</revision>
+        <changelist>-SNAPSHOT</changelist>
+        <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
         <jenkins.version>2.387.3</jenkins.version>
         <bom>2.387.x</bom>
-        <bom.version>2423.vce598171d115 </bom.version>
-        <disabledTestInjection>true</disabledTestInjection>
+        <bom.version>2543.vfb_1a_5fb_9496d</bom.version>
         <hpi.compatibleSinceVersion>20.2.34</hpi.compatibleSinceVersion>
     </properties>
 
     <name>Fortify</name>
     <description>Adds the ability to perform security analysis with Fortify Static Code Analyzer, upload results to Software Security Center, show analysis results summary, and set build failure criteria based on analysis results.</description>
-    <url>https://github.com/jenkinsci/fortify-plugin</url>
+    <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
```

Comment:
As per latest recommendations in the archetype: https://github.com/jenkinsci/archetypes